.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "tabwidth": 2,
+  "useTabs": false
```

Comment:
These are defaults and we don't plan to change the defaults anyway. Doesn't it work without this file?

Reply:
Some people have different prettier configuration (like me), and this will force their editor to use these settings.

Reply:
Oh, interesting. Does having an empty config in `package.json` help?